### ISSUE TITLE
Glosses containing 'or'

### ISSUE THREAD

**Issue by jmccrae** [globalwordnet/english-wordnet#109]
Many glosses seem to refer to two distinct concepts, e.g.,

10148446-n (Interlingual Index: i90200)
(n) monster, behemoth, colossus, giant, goliath 
> someone or something that is abnormally large and powerful

These should probably be split into new synsets

**Comment by arademaker** [comment]
I agree. Almost all cases of OR in the glosses cause confusion. But they are so frequent that a second thought about the implications of this proposal is needed. This should not be made case-by-case.
